lib/facade/MonitoringAspect.ts

```diff
@@ -0,0 +1,55 @@
+import { IConstruct, IAspect } from "monocdk";
+import * as apigw from "monocdk/aws-apigateway";
+import * as apigwv2 from "monocdk/aws-apigatewayv2";
+import * as acm from "monocdk/aws-certificatemanager";
+
+import { MonitoringFacade } from "./MonitoringFacade";
+
+export interface MonitoringAspectProps {
```

Comment:
Later we can also add `defaultAcmMonitoringProps?: AcmMonitoringProps` that would be applied to all such constructs. That way, we could add automatic alarms, and so on.